From b1dd9b22ad061dfe6fe66928e583b92a7819e628 Mon Sep 17 00:00:00 2001
From: Ren Chenglei <chenglei.ren@intel.com>
Date: Tue, 13 Jun 2023 17:48:20 +0530
Subject: [PATCH] INTERNAL: Yf tiled and Yf tiled CCS should be set Y Tile

As Yf tiled and Yf tiled CCS are not set to any tile,
scramble screen issues seen.

Fix the issue by setting Yf tiled and Yf tiled CCS to Y tile.

Signed-off-by: Chenglei Ren <chenglei.ren@intel.com>
---
 i915.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/i915.c b/i915.c
index c41cbb8..9399067 100644
--- a/i915.c
+++ b/i915.c
@@ -653,6 +653,10 @@ static int i915_bo_compute_metadata(struct bo *bo, uint32_t width, uint32_t heig
 	 * IPs(render/media/display)
 	 */
 	case I915_FORMAT_MOD_Y_TILED_GEN12_RC_CCS:
+#ifdef USE_GRALLOC1
+	case I915_FORMAT_MOD_Yf_TILED:
+	case I915_FORMAT_MOD_Yf_TILED_CCS:
+#endif
 		bo->meta.tiling = I915_TILING_Y;
 		break;
 	case I915_FORMAT_MOD_4_TILED:
-- 
2.40.1

